analytics/views.py

```diff
@@ -1,3 +1,30 @@
 # from django.shortcuts import render
+from django.utils.decorators import decorator_from_middleware
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from . import middleware, models, serializers
 
 # Create your views here.
+
+
+@api_view(['GET'])
+def home(_request):
+    """Render the most basic home page ever designed."""
+    return Response('Analytics app seems to be running.')
+
+
+@api_view(['GET', 'POST'])
+@decorator_from_middleware(middleware.AnalyticsTokenAuthMiddleware)
+def event_list(request):
```

Comment:
A może można użyć widoków generycznych?

https://www.django-rest-framework.org/api-guide/generic-views/

Reply:
Można, zrobiłem inaczej, bo było zwięźlej. Skoro jest sugerowane, to 0d50b6b4f93ab0434a462de27202d0ae66f81e6d.